src/Native/System.Native/pal_networking.cpp

```diff
@@ -218,11 +209,7 @@ extern "C" int32_t GetHostEntryForName(const uint8_t* address, HostEntry* entry)
         return ConvertGetAddrInfoAndGetNameInfoErrorsToPal(result);
     }
 
-    *entry = {
-        .CanonicalName = nullptr,
-        .AddressListHandle = reinterpret_cast<void*>(info),
-        .IPAddressCount = 0
-    };
+    *entry = {.CanonicalName = nullptr, .AddressListHandle = reinterpret_cast<void*>(info), .IPAddressCount = 0};
```

Comment:
I believe it is this rule ```ConstructorInitializerAllOnOneLineOrOnePerLine``` which we have set to ```true``` to enforce rules around constructors. Initializer lists are old C-style things that are disabled via clang by default and throw a warning so we should think about if we want to use them or not. In either case, this pack is more about running the formatting tool to bring us into style-conformance rather than changing the style; I'd be more than happy to change the style if we so choose, but I think we should do that in a different pack. 

Thoughts?